repooler.py

```diff
@@ -297,15 +297,18 @@ def integrate_conc_diff(lane_maps, desired_ratios):
         for index in xrange(1, len(lane_maps[key])):
             expect_output.append(1/float(len(lane_maps[key]) -1))#-1 Removes undetermined
         #TODO: One could include undetermined here
-        conc_factor[key] = actual_output/expect_output
+        #Overriding errstate since expected is 0 for undetermined
+        with numpy.errstate(divide='ignore'):
+            conc_factor[key] = actual_output/expect_output
         #Division verified. One with high Actual/Expressed should naturally be less of in repool.
+        
         volume_ratios[key] = desired_ratios[key] / conc_factor[key]
         #Merge down to 100%. Makes sense, since samples differ in volume.
         if not sum(volume_ratios[key]) == 0:
             volume_ratios[key] = volume_ratios[key]/sum(volume_ratios[key])
         else: 
             # Should fix NAN bug
-            volume_ratios[key] = 0
+            volume_ratios[key] = numpy.zeros(len(volume_ratios[key]))
```

Comment:
The comment is a bit misleading and I should probably change it. What is actually resolves is the case where a lane structure has no samples that need to be repooled. The samples in the lane structure still need to be accounted for, which is why I don't simply omit them from the program.